ethcore/snapshot/src/service.rs

```diff
@@ -265,6 +265,7 @@ pub struct Service<C: Send + Sync + 'static> {
 	client: Arc<C>,
 	progress: RwLock<Progress>,
 	taking_snapshot: AtomicBool,
+	taking_snapshot_num: AtomicUsize,
```

Comment:
I would call it `taking_snapshot_at`, otherwise it read as if it's a number of snapshots we're taking